Added the application form

RachelMuston opened the pull request:
The link to the application form is active in the user guide and is set up so that the file automatically downloads on click.

The link to the application form is active in the 'how-to-apply' footer and currently it is set up so that file opens in the browser.  This is because as far as I can tell Liquid doesn't allow a direct download link (only way around it is to put the .pdf in a .zip file, or not put the .pdf file up and instead put up the .odt file)?

Thoughts @smellems @ElminaIusifova?

smellems reviewed (commented):
I think linking to the PDF is good.

I noticed that there is no character limit for the Opportunity name, Supplier name and Business name fields.  Maybe we should have one..
I think those fields should all be the same height.

Steps I did:
1. Right-click the link and open in a new tab (I always do that)
2. PDF open in a new tab in my browser
3. Fill in the fields in the PDF
4. Save PDF
5. Open saved PDF
6. Information I put in fields is still there :tada:

RachelMuston commented:
Oh interesting, thanks Seb!  If we are ok with the form opening in the browser, I'm wondering if for consistency, we should have the link in the supplier guide also open up in the browser (rather than being a download link).  I'm also thinking that we should force the form to open in a new tab (was good to know that is your standard behaviour anyway - that's what got me thinking).

RachelMuston commented:
Ok I think i've fixed everything! For opp name, supplier name and business name field, I made those fields all the same height, added a multi-line option (in case the text doesn't fit in the box), I also increased the font size for the boxes to 12 and I added a character limit to all of those fields.
I also adjusted the footer and the user guide so that the form opens in a new tab.
@smellems if you could check to make sure that is all working for you, that would be awesome! :)

Oh and I also added some additional text at the bottom of the form to remind suppliers that by applying to an opportunity, they agree to do the work by the delivery date and be paid by credit card/paypal.

ElminaIusifova commented:
I just saved the form locally and tried to fill it out and save it again. It worked perfectly for me. Approving PR
